baseapp/baseapp.go

```diff
@@ -361,7 +361,7 @@ func (app *BaseApp) setCheckState(header ocproto.Header) {
 
 	app.checkState = &state{
 		ms:  ms,
-		ctx: ctx.WithConsensusParams(app.GetConsensusParams(ctx)),
+		ctx: ctx,
```

Comment:
This modification reverts a part of https://github.com/line/lbm-sdk/pull/142.
The previous code causes sim test failure.